src/network/ConnectionForward.ts

```diff
@@ -239,8 +239,9 @@ class ConnectionForward extends Connection {
 
   protected startTimeout() {
     this.timeout = setTimeout(() => {
+      this.logger.warn('ConnectionForward timing out');
       this.tlsSocket.emit('error', new networkErrors.ErrorConnectionTimeout());
-    }, this.timeoutTime);
+    }, 1000);
```

Comment:
We should probably have separate branches if we are going to be doing some temporary messing around.

Reply:
This is my bad. Feel free to revert my WIP commit on this branch. They're the only temp changes I made.